dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryAPI.java

```diff
@@ -176,9 +176,20 @@
 	 * @throws DotSecurityException 
 	 */
 	public void save(Category parent, Category object, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
-	
+
+    /**
+     * Important: should be use only for Push Publish.
+     *
+     * Save Remote categories, in this case we don't need to check if this category exists into
+     * the system but just save with its own inode.
+     *
+     */
+    void saveRemote(Category parent, Category object, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)